### PR TITLE
Tutorials for test cross and sibling pair challenges

### PR DESCRIPTION
**Pull request by dstrawberrygirl**

### CODE REVIEW

File: src/code/modules/tutorial.json [hunk 1]
```diff
@@ -71,6 +71,113 @@
...
+        {
+          "mission": 0,
+          "challenges":[
+            {
+              "challenge": 0,
+              "steps":[
+                {
+                  "element": "father",
+                  "text": "Parent Drake",
+                  "more": "This drake’s alleles are invisible!",
```
Comment (kswenson) [inline src/code/modules/tutorial.json]: I would change "invisible" to "hidden". The former makes it seems like a property of the drake, as though it's a stealth drake.

File: src/code/modules/tutorial.json [hunk 2]
```diff
@@ -71,6 +71,113 @@
...
+                  "more": "This is the breeder's name for the drake color."
+                }
+              ]
+            }
+          ]
+        }
+      ]
+    },
+    {
+      "level": 3,
```
Comment (kswenson) [inline src/code/modules/tutorial.json]: First of two entries for this level.

**Comment by kswenson** [comment]
I couldn't get some of the tutorials to appear. Multiple entries for the same level seem to confuse it.

File: src/code/modules/tutorial.json [hunk 3]
```diff
@@ -71,6 +71,113 @@
...
+                  "more": "Click Breed to produce offspring. Then click a baby drake that matches the target!"
+                }
+              ]
+            }
+          ]
+        }
+      ]
+    },
+    {
+      "level": 3,
```
Comment (kswenson) [inline src/code/modules/tutorial.json]: Second entry for this level. The code ignores multiple entries per level.

**Comment by kswenson** [comment]
Two other thoughts not directly related to this PR:
1. We should have a schema for `tutorial.json`
2. It should probably live in the `resources` folder

File: src/code/modules/tutorial.schema.json [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+                    "properties": {
+                      "steps": {
+                        "description": "The list of tutorial steps for this challenge",
+                        "type": "array",
+                        "items":{
+                          "element": "The css class of the element being highlighted",
+                          "text": "The main header text to show on the popup",
+                          "more": "The full text of the tutorial displayed on clicking More",
+                          "pulseElement": "The element to highlight with a css pulse (if different from the main element",
+                          "location": "A specific location on screen for the popup"
```
Comment (kswenson) [inline src/code/modules/tutorial.schema.json]: The items should be in the form:
```
"items": {
  "description": "A tutorial step",
  "type": "object",
  "properties": {
    "element": {
      "description": "The css class of the element being highlighted",
      "type": "string",
    },
    ...
```

Comment (dstrawberrygirl) [reply src/code/modules/tutorial.schema.json]: Got it, thanks - figured I'd need a second pass at the schema!

**Comment by dstrawberrygirl** [comment]
The third tutorial of the test cross doesn't have a visible offspring drake to highlight so I have opted to pulse the clutch counter for now. I'll be reaching out for feedback on the best approach here - there is no capacity currently in the tutorial system to show a hint at a different stage of a specific challenge.

**Pull request merged**